Phase01/Phase01-StudentAverage/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Phase01_StudentAverage
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            string studentsFilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Assets", "Students.json");
+            string coursesFilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Assets", "Courses.json");
```

Comment:
move file paths to resource (.resx) file

Reply:
solved.